### PR TITLE
OCPBUGS-7874: set the pause exec path for openshift pod image

### PR DESCRIPTION
**Pull request by zshi-redhat**
openshift pod image places pause exec under /usr/bin/pod while the default path for pause exec is set to "/pause".

### CODE REVIEW

**Comment by dhellmann** [comment]
@mrunalp @haircommander  could I get one of you to take a look at these changes for us, please?

File: packaging/crio.conf.d/microshift_amd64.conf [hunk 1]
```diff
@@ -2,8 +2,19 @@
...
 cgroup_manager = "systemd"
 default_runtime = "crun"
+default_sysctls = [
+    "net.ipv4.ping_group_range=0 2147483647",
+]
+manage_ns_lifecycle = true
+absent_mount_sources_to_reject = [
+    "/etc/hostname",
+]
+drop_infra_ctr = true
```
Comment (ggiguash) [inline packaging/crio.conf.d/microshift_amd64.conf:17]: Can we add a comment in the file on why these settings are added? There's nothing in the description of the PR.

Comment (haircommander) [inline packaging/crio.conf.d/microshift_amd64.conf:17]: this value is the default, why are we specifying manually?

Comment (zshi-redhat) [reply packaging/crio.conf.d/microshift_amd64.conf]: I copied it from openshift crio config: https://github.com/openshift/machine-config-operator/blob/cbd7d9514d6a03c84192058edae7a6ff4f9d6941/templates/master/01-master-container-runtime/_base/files/crio.yaml#L34

removed it in the new PR version.

File: packaging/crio.conf.d/microshift_amd64.conf [hunk 2]
```diff
@@ -2,8 +2,19 @@
...
 conmon_cgroup = "pod"
+default_env = [
+    "NSS_SDB_USE_CACHE=no",
+]
 cgroup_manager = "systemd"
 default_runtime = "crun"
+default_sysctls = [
+    "net.ipv4.ping_group_range=0 2147483647",
+]
+manage_ns_lifecycle = true
```
[the remark this answers is not in the document]
Comment (zshi-redhat) [reply packaging/crio.conf.d/microshift_amd64.conf]: removed.

File: packaging/crio.conf.d/microshift_amd64.conf (hunk 1 above)
Comment (zshi-redhat) [reply packaging/crio.conf.d/microshift_amd64.conf]: I added the comment in the commit message.

**Comment by haircommander** [comment]
/lgtm

thank you!

**Pull request merged**